migration/sql-files/012-hide-email.sql

```diff
@@ -0,0 +1,3 @@
+-- Store identity provider username for the external token
+ALTER TABLE users ADD COLUMN email_private boolean;
+UPDATE USERS SET email_private = false;
```

Comment:
Maybe set it to NOT_NULL after updating the table?

Reply:
7c096653afb69415af9e1ad559255cddbfba6e04